packages/es-components/src/components/controls/buttons/ToggleButton.js

```diff
@@ -9,6 +9,12 @@ import { useTheme } from '../../util/useTheme';
 function ToggleButton(props) {
   const theme = useTheme();
   const [isPressed, setIsPressed] = useState(props.isPressed);
+  const prevPressedProp = useRef(props.isPressed);
```

Comment:
👍 looks like a good solution here. glad to see you were able to work it out.

Reply:
It does restore the old [`componentWillReceiveProps`](https://github.com/WTW-IM/es-components/blob/3923764e36de07dc4130ddcc268f8020ce052e34/packages/es-components/src/components/controls/buttons/ToggleButton.js#L13-L17) functionality. I'm still debating whether or not `Menu` really needs to use the `ToggleButton`.

Reply:
I think it's fine. I would rather keep it as is if it's working okay. this extra little bit of code isn't going to hurt anything when rendering.